api/pom.xml

```diff
@@ -53,7 +53,7 @@
       </dependency>
       <dependency>
          <groupId>org.apache.velocity</groupId>
-         <artifactId>velocity</artifactId>
+         <artifactId>velocity-engine-core</artifactId>
```

Comment:
Did the artifact id change?

Reply:
yes @dkayiwa  it changed according to this http://velocity.apache.org/download.cgi#engine